### DISCUSSION TITLE
ylt的struct_pack编解码兼容性

### DISCUSSION THREAD

**Discussion by bolanbujing** [alibaba/yalantinglibs#835] [Announcements]
前后兼容性，比如0.2.8版本和0.3.10版本之间的编解码，不同系统，比如win和linux上的编解码，兼容性有保证吗

**Comment by poor-circle** [comment]
我们保证不同版本，不同系统，不同架构（大小端）上的兼容性。如果你有发现兼容性bug可以反馈。
